lua/asllib.lua

```diff
@@ -15,18 +15,19 @@ function clamp( input, min, max )
 end
 
 function note( noteNum, duration )
-    return{ to( n2v(noteNum), 0 )
-          , to( 'here', duration  )
-          }
+    return to( n2v(noteNum), duration, 'now' )
```

Comment:
should this be `return{ to( n2v(noteNum), duration, 'now' ) }` ?

Reply:
This is fine- a standalone `to()` is a valid ASL by itself. You only need the table if you have more than one action. `to` is similar to the other constructs like `loop`, ie they are just functions that return ASLs `loop{ to(), to() }`.

Sorry I still haven't done the full write up of ASL :(